src/ui/public/field_format_editor/pattern/pattern.html

```diff
@@ -1,5 +1,6 @@
 <div class="form-group">
   <input
+    id="{{ id }}"
```

Comment:
It corresponds to labels used outside of the directive: https://github.com/elastic/kibana/pull/13364/files#diff-cdc9adaaaf6e3096b1edb04fc611d045